docs/offline.md

```diff
@@ -0,0 +1,11 @@
+# @extends
+
+## Offline app #target-app
+
+The MakeCode editor is available as app which you can install on a computer with Windows or Mac OS. Once installed, the **[MakeCode Offline App](/offline-app)** lets you create, run, and download your projects to the @boardname@. It works the same as the Web application does in your browser but it's a stand-alone application that will work when a connection to the internet is restricted or not available.
+
+### ~ hint
+
+Click the link here: [MakeCode Offline App](/offline-app)
```

Comment:
```suggestion
### ~button /offline-app

Download the offline app!
```

that will make it a button:

![image](https://user-images.githubusercontent.com/5615930/92050051-f148ee80-ed40-11ea-83c9-6a44bdcaee63.png)